finance-tracker/app/controllers/friendships_controller.rb

```diff
@@ -0,0 +1,11 @@
+class FriendshipsController < ApplicationController
+  def create
```

Comment:
Whats this for?
If no use, remove it and don't forget to remove it from route as well

Reply:
it will be used in the next PR

Reply:
Next time, have it in the next PR instead of here